friends/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import axios from 'axios';
+import Friends from './components/Friends';
+import './App.css';
+
+class App extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      friends: []
+    }
+  }
+
+  //setState to friends: res.friends
+  componentDidMount() {
+    axios
+    .get("http://localhost:5000/friends")
+    .then(resolved => {
```

Comment:
'res' stands for 'response', not 'resolved'.

We also use 'res' because it's faster to type out.

Reply:
Thanks!